lectures/heavy_tails.md

```diff
@@ -206,6 +242,12 @@ The histogram also looks different to the histogram of the normal
 distribution:
 
 ```{code-cell} ipython3
+---
+mystnb:
+  figure:
+    caption: Histogram (Normal vs Bitcoin returns)
```

Comment:
@jstac I have used capitals for `Normal` and `Bitcoin`. Is that correct style?

Reply:
I usually follow Wikipedia.

Reply:
Thanks @jstac I see. I have changed Normal to normal in text and kept Cauchy and Pereto (as names) capitalised as per wikipedia.